script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt

```diff
@@ -659,7 +674,7 @@ Even my millennia of love may fade at such a sight."
 -- Page 169, Offset 39900.
 -- 『出力はあがろうと、欠点が多くなればそれは弱体化だよ。
 %{i}
-"Even if the output power rises,
+"Even if her power is magnified,
  if the number of flaws increases,
  then she has weakened.
```

Comment:
Still don't fully jive with this sentence.
I think I suggested more extensive changes, but regardless.
>Even if her power is magnified,
>when her flaws increase with it, 
>that would make her weaker.